src/NzbDrone.Core/Indexers/NyaaTorrentRssParser.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Linq;
+using System.Xml.Linq;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Indexers.Exceptions;
+
+namespace NzbDrone.Core.Indexers
+{
+    public class NyaaTorrentRssParser : TorrentRssParser
+    {
+        public NyaaTorrentRssParser()
+        {
+        }
+
+        protected override long GetSize(XElement item)
+        {
+            var itemstripped = item.StripNameSpace();
+            if (itemstripped.Element("size") != null)
+            {
+                return ParseSize(itemstripped.Element("size").Value, true);
+            }
+
+            return base.GetSize(item);
+        }
+
+        protected override string GetInfoHash(XElement item)
+        {
+            var itemstripped = item.StripNameSpace();
+            if (itemstripped.Element("infoHash") != null)
+            {
+                return (itemstripped.Element("infoHash").Value);
+            }
+
+            return base.GetInfoHash(item);
+        }
+
+        protected override int? GetSeeders(XElement item)
```

Comment:
Similar to `InfoHashElementName` add one for seeders and use that before parsing from description.

Reply:
Alternatively, pull the EzssTorrentRssParser GetSeeders function to the parent. I t hink it's safe to always use the element if it's present (and valid), and fall back to description if ParseSeedersInDescription is enabled.

Reply:
Was actually able to pull most of Ezrss to the parent now with some reworking;  pushed it up